### PR TITLE
add options to load input from sdp file and to export resulting file

### PR DESCRIPTION
**Pull request by kerautret**

### CODE REVIEW

File: volumetric/homotopicThinning3D.cpp [hunk 1]
```diff
@@ -80,9 +81,12 @@ int main( int argc, char** argv )
   general_opt.add_options()
     ( "help,h", "display this message." )
     ( "input,i", po::value<std::string>(), "Input volumetric file (.vol, .pgm3d or p3d)" )
-    ( "min,m", po::value<int>()->default_value( 0 ), "Minimum (excluded) value for threshold." )
-    ( "max,M", po::value<int>()->default_value( 255 ), "Maximum (included) value for threshold." )
-    ("fixedPoints", po::value<std::vector <int> >()->multitoken(), "defines the coordinates of points which should not be removed." );
+    ( "fixedPointSDP,s", po::value<std::string>(), "use fixed points from a file instead.")
```
Comment (dcoeurjo) [inline volumetric/homotopicThinning3D.cpp]: remove instead

Comment (dcoeurjo) [inline volumetric/homotopicThinning3D.cpp]: move this option below ```fixedPoints``` one

File: volumetric/homotopicThinning3D.cpp [hunk 2]
```diff
@@ -143,8 +147,13 @@ int main( int argc, char** argv )
       trace.error()<< " The coordinates should be 3d coordinates, ignoring fixedPoints option." << std::endl;
     }
   }
-
-
+  if(vm.count("fixedPointSDP")){
+    std::vector<Z3i::Point> vPt = PointListReader<Z3i::Point>::getPointsFromFile(vm["fixedPointSDP"].as<std::string>());
+    for( auto &p: vPt){
```
Comment (dcoeurjo) [inline volumetric/homotopicThinning3D.cpp]: please check you IDE to match with DGtal parenthesis/indent conventions.  ```{``` in new lines for instance.

**Comment by dcoeurjo** [comment]
Fine for me too, you can merge.
thanks

File: volumetric/homotopicThinning3D.cpp (hunk 1 above)
Comment (kerautret) [inline volumetric/homotopicThinning3D.cpp]: yes thanks

**Comment by kerautret** [comment]
thanks for the remarks I merge

**Pull request merged**